### PR TITLE
Run yarn build:docs:integrations

### PR DESCRIPTION
**Pull request by ndowmon**
Not urgent. Just ran the script to pull some changes.

### CODE REVIEW

File: docs/integrations/microsoft-365/index.md [hunk 1]
```diff
@@ -54,27 +57,30 @@ If you need help with this integration, please contact
    needs. You may leave this as `DISABLED` and manually execute the integration.
 7. Click **Create Configuration** once all values are provided.
 
-   ![Create Configuration](../../../assets/m365CompleteConfiguration.png)
-   ![Create Configuration](../../../assets/m365IntegrationCreatedDialog.png)
+   <img src="./images/CompleteConfiguration.png" alt="drawing" width="200"/>
+   <img src="./images/IntegrationCreatedDialog.png" alt="drawing" width="200"/>
```
Comment (ndowmon) [inline docs/integrations/microsoft-365/index.md:61]:
```suggestion
   ![Create Configuration](../../../assets/m365CompleteConfiguration.png)
   ![Create Configuration](../../../assets/m365IntegrationCreatedDialog.png)
```

File: docs/integrations/microsoft-365/index.md [hunk 2]
```diff
@@ -54,27 +57,30 @@ If you need help with this integration, please contact
...
+   <img src="./images/OffsiteDialog.png" alt="drawing" width="200"/>
+
+9. You will be directed to Microsoft's identity platform where you must login in
+   as a global administrator of the organizational Active Directory tenant you
+   intend to integrate with.
 
-9. You will be directed to Microsoft's identity platform, where you must login
-   in as an administrator in the Microsoft organization you intend to integrate
-   with.
+      <img src="./images/PickAnAccount.png" alt="drawing" width="200"/>
```
Comment (ndowmon) [inline docs/integrations/microsoft-365/index.md:71]:
```suggestion
      ![directed to Microsoft's identity platform](../../../assets/m365PickAnAccount.png)
```

File: docs/integrations/microsoft-365/index.md [hunk 3]
```diff
@@ -54,27 +57,30 @@ If you need help with this integration, please contact
...
+   NOTE: You must select an account belonging to an organizational tenant. When
+   you are already logged into an account, the badge icons indicate the nature
+   of the tenant the account belongs to.
 
-   NOTE: You must select a business account which are indicated by badge icons.
-   If you select a personal account, you will recieve the below error messaage.
+   Selecting a personal account produces the following error message.
 
-      ![select a business account](../../../assets/m365PersonalAccountError.png)
+      <img src="./images/PersonalAccountError.png" alt="drawing" width="200"/>
```
Comment (ndowmon) [inline docs/integrations/microsoft-365/index.md:79]:
```suggestion
      ![select a business account](../../../assets/m365PersonalAccountError.png)
```

File: docs/integrations/microsoft-365/index.md [hunk 4]
```diff
@@ -54,27 +57,30 @@ If you need help with this integration, please contact
...
 
 10. Review requested permissions (which are described below) and grant consent.
 
-      ![Review requested permissions](../../../assets/m365PickAnAccount.png)
+      <img src="./images/PickAnAccount.png" alt="drawing" width="200"/>
```
Comment (ndowmon) [inline docs/integrations/microsoft-365/index.md:83]:
```suggestion
      ![Review requested permissions](../../../assets/m365PickAnAccount.png)
```

**Review by ndowmon — COMMENTED** [review]
Reverting the HTML image links to use those custom Markdown image links.

**Review by aiwilliams — APPROVED** [review]
🙇

**Pull request merged**